packages/babel-plugin-transform-parameters/test/fixtures/parameters/default-rest-mix/expected.js

```diff
@@ -3,9 +3,9 @@ function fn(a1) {
   var _arguments$ = arguments[2],
```

Comment:
Shouldn't this also be `arguments.length > 2 ? arguments[2] : undefined`?

Reply:
Mh, I guess it isn't needed because if `arguments.length > 2` the following line is going to throw, so optimizing for that case it is unuseful... Is it correct?

Reply:
That means that a single invocation where this function throws is enough to turn the property accesses into slow mode for the rest of the programs execution. We should try to avoid this.

Reply:
Property access in general or just `arguments[...]` in this function?

Reply:
This particular `arguments[2]` property access.